src/SUMMARY.md

```diff
@@ -21,3 +22,37 @@
   - [Network]()
     - [journées_portes_ouvertes](./Phack2021/Network/journées_portes_ouvertes/README.md)
     - [Système_nom_de_domaine](./Phack2021/Network/Sys_nom_domaine/README.md)
+  - [Cracking]()
+    - [Cracky](./Phack2021/Cracking/Cracky/README.md)
+    - [Etsy](./Phack2021/Cracking/Etsy/README.md)
+    - [Sammy](./Phack2021/Cracking/Sammy/README.md)
+  - [Cryptography]()
+    - [Certifié_sécurisé](./Phack2021/Cryptography/Certifié_sécurisé/README.md)
+    - [Enchaîné](./Phack2021/Cryptography/Enchaîné/README.md)
+    - [Guacamole](./Phack2021/Cryptography/Guacamole/README.md)
+    - [H3lp](./Phack2021/Cryptography/H3lp/README.md)
+  - [Programming]()
+    - [A-Maze-Ing](./Phack2021/Programming/A-Maze-Ing/README.md)
+    - [Ben_&_Harry](./Phack2021/Programming/Ben_&_Harry/README.md)
+    - [Quick_Response_Code](./Phack2021/Programming/Quick_Response_Code/README.md)
+    - [WikiBot](./Phack2021/Programming/WikiBot/README.md)
+  - [Social_Engineering]()
+    - [Agent_Secrétariat](./Phack2021/Social_Engineering/Agent_Secrétariat/README.md)
+  - [Steganography]()
+    - [Aliens](./Phack2021/Steganography/Aliens/README.md)
+    - [Alter_Egg-o](./Phack2021/Steganography/Alter_Egg-o/README.md)
+    - [Caumunikassion](./Phack2021/Steganography/Caumunikassion/README.md)
+    - [Chasse_aux_oeufs](./Phack2021/Steganography/Chasse_aux_oeufs/README.md)
+    - [Strong_Daddy](./Phack2021/Steganography/Strong_Daddy/README.md)
+    - [Une_douce_petite_musique](./Phack2021/Steganography/Une_douce_petite_musique/README.md)
+  - [System]()
+    - [Sudoku](./Phack2021/System/Sudoku/README.md)
+    - [To_B,or_!_to_B](./Phack2021/System/To_B,or_!_to_B/README.md)
```

Comment:
It seems special chars are not the best inside the file path, mdbook can't find the correct file